src/processing.py

```diff
@@ -105,34 +186,123 @@ def process_cog(task: QueueTask):
         runtime=t2 - t1,
         user_id=task.user_id,
         compression=options.profile,
-        overviews=options.overviews,
-        # !! This is not correct!! 
+        overviews=overviews,
+        tiling_scheme=options.tiling_scheme,
         resolution=int(options.resolution * 100),
```

Comment:
@cmosig sorry i accidentally deleted your comment. The resolution here is only the provides maximum resolution of the processing. We need to implement the native resolution of the image separately #72